charts/astronomer/templates/commander/commander-deployment.yaml

```diff
@@ -158,7 +160,7 @@ spec:
               value: "true"
             {{- end }}
             - name: COMMANDER_MANAGE_NAMESPACE_RESOURCE
-              value: "false"
+              value: {{ if .Values.global.rbacEnabled }}"true"{{ else }}"false"{{ end }}
```

Comment:
@pgvishnuram this was set to `false` in 1.0. Will enabling this cause more code paths to turn on? Will we need additional testing because of this change?

Reply:
houston api is not impacted with this change - no code path being affected on this change